function/index.test.js

```diff
@@ -19,6 +19,12 @@ describe('Kofax scanning processorfunction', () => {
         const response = await handler({}, null);
         expect(response).toContain('Success!');
     });
+
+    it('Should error if there are no messages to consume', async () => {
```

Comment:
Can you just update the description of this test - I don't think we are expecting an error, I think we're just expecting it resolve successfully to 'Nothing to process'.